recipe/meta.yaml

```diff
@@ -1,35 +1,35 @@
 {% set name = "weasel" %}
-{% set version = "0.3.4" %}
+{% set version = "0.4.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: eb16f92dc9f1a3ffa89c165e3a9acd28018ebb656e0da4da02c0d7d8ae3f6178
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: aabc210f072e13f6744e5c3a28037f93702433405cd35673f7c6279147085aa9
 
 build:
   number: 0
   script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
   entry_points:
     - weasel = weasel.cli:app
+  skip: true  # [py<37]
 
 requirements:
   host:
     - python
     - pip
     - setuptools
-    - wheel
```

Comment:
Same question here

Reply:
Because recent versions of setuptools are PEP 517 compliant (i.e. they can generate wheels). 
https://peps.python.org/pep-0517/

The next linter release will remove this check. And I removed it because recipe-bumper-cli removed it.